tools/init.js

```diff
@@ -83,6 +83,11 @@ console.log(colors.yellow('Hi! I\'m setting things up for you!!'))
 rm('-Rf', path.resolve(__dirname, '..', '.git'))
 console.log('\r\n', 'Removed .git directory','\r\n')
 
+// Remove files
+const files_rm = ['tools/init.js']
+const paths_rm = files_rm.map((f) => path.resolve(__dirname, '..', f))
+rm(paths_rm)
+console.log(`\r\nRemoved files: ${files_rm.toString()}\r\n`)
 
```

Comment:
Note I've used ES6 syntax in 2 lines - thus will nto work with older version of nodejs